CHANGES.md

```diff
@@ -14,8 +14,10 @@
 
 ### Changes
 
+* remove `pyaml` python dependency.
 
 ### Upgrade Notes
+
```

Comment:
```suggestion
You will need to update the version of sel4-deps installed
```

Something along the lines of this?

Reply:
It will work with older versions of `sel4-deps` -- the more recent `sel4-deps` version installs strictly fewer packages.

Reply:
Oh my bad, yes right I thought that `yaml` wasn't available for direct import without the update.